Unreal/CarlaUE4/Plugins/CarlaTools/Source/CarlaTools/Private/ProceduralBuildingUtilities.cpp

```diff
@@ -232,8 +238,15 @@ UMaterialInstanceConstant* AProceduralBuildingUtilities::GenerateBuildingMateria
   const FString PackageFileName = FPackageName::LongPackageNameToFilename(
       PackageName, 
       ".uasset");
-  UPackage::SavePackage(NewPackage, DuplicatedMaterial, EObjectFlags::RF_Public | EObjectFlags::RF_Standalone,
-      *PackageFileName, GError, nullptr, true, true, SAVE_NoError);
+  FSavePackageArgs SaveArgs;
```

Comment:
This code is repeated two times in the same file. Consider extract this code to a function.